### ISSUE TITLE
Has anyone successfully trained a Flux style lora e.g. clothing?

### ISSUE THREAD

**Issue by PheonixAi420** [derrian-distro/LoRA_Easy_Training_Scripts#286]
I've only ever been able to do character loras and wanting to do some styles.

**Comment by Jelosus2** [comment]
If you were able to train a character you should be able to train other things like clothing, concepts or art styles too. Being able to train something successfully depends on the base model and/or your dataset and not the training scripts used.

**Comment by PheonixAi420** [comment]
I tried to make a clothing style but it affected the face of the character lora i was using. I tried using face crop but idk what I'm doing with that. I've probably made over 200 character/ celebrity loras and was trying to make a certain type of clothing that isn't in the training sets of the character loras. I didn't see a place in the UI to tell it whether its a style, concept or style etc. Am I missing something?

**Comment by Jelosus2** [comment]
What you want to train is based on the dataset you have not the settings in the UI

**Comment by PheonixAi420** [comment]
Hmm alright. It just always seems to use the faces that are in the dataset and train them as well which effects the character Lora's face and was just trying to see if there was a way to make it only learn the clothing style and not train the faces.

**Comment by Jelosus2** [comment]
There's something called masked training that only learns what is masked but not sure if that works for flux

**Comment by PheonixAi420** [comment]
Alright well thanks for the info If I figure it out I will let you know that way if anybody else asks you can tell them.

**Comment by PheonixAi420** [comment]
@Jelosus2 Yeah I dont think masked training works, it's greyed out and cant find a way to un-grey it. It seems it's not possible like it was for SDXL. Idk.
